frontend/src/components/pages/SearchReviews/SearchReviews.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Center } from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+
+import { Review } from "../../../types/ReviewTypes";
+import LoadingSpinner from "../../common/LoadingSpinner";
+import SearchBox from "../SearchBox";
+import mockReviews from "./mockReviews";
+import ReviewsGrid from "./ReviewsGrid";
+
+/**
+ * The component for the page where the user searches and filters reviews.
+ */
+const SearchReviews = (): React.ReactElement => {
+  const [loading, setLoading] = useState<boolean>(false);
+  const [searchText, setSearchText] = useState<string>("");
+  const [genresFilter, setGenresFilter] = useState<string[]>([]);
+  const [ageRangeFilter, setAgeRangeFilter] = useState<number[]>([]); // ageRange[0] is min age, ageRange[1] is max age
+  const [displayedReviews, setDisplayedReviews] = useState<Review[]>([]);
+
+  /** Fetches the search queries from url */
+  useEffect(() => {
+    const params = new URLSearchParams(window.location.search);
```

Comment:
Could you document these search params? (ie what each of them are, what they do, some examples, etc). You could maybe put it in the component header